cookbooks/fb_syslog/recipes/default.rb

```diff
@@ -74,6 +74,12 @@
   notifies :restart, "service[#{service_name}]"
 end
 
+directory '/etc/rsyslog.d' do
+  action :delete
+  recursive true
+  notifies :restart, "service[#{service_name}]"
+end
+
```

Comment:
Let's do this in another PR please, and ideally put it behind an attribute gate.